knowledge_repo/app/routes/editor.py

```diff
@@ -362,7 +363,7 @@ def s3_upload(file_name, file_data):
                 s3_client,
                 html_data,
                 AWS_S3_BUCKET,
-                html_file_name,
+                path + '.kp/' + html_file_name,
```

Comment:
Use `os.path.join()` as explained in: https://www.geeksforgeeks.org/python-os-path-join-method/ ?